Packs/SplunkPy/Integrations/SplunkPy/SplunkPy.py

```diff
@@ -373,7 +375,7 @@ def fetch_notables(service, mapper, cache_object=None, enrich_notables=False):
                     'query: {} is: {}.'.format(last_run_time, now, fetch_query, len(incidents))
     extensive_log(debug_message)
 
-    if not enrich_notables:
+    if not enrich_notables or not cache_object:
```

Comment:
what will be here if this are the first fetch and the enrichment is enabled ?
cache_object will not be None?